lib/commands/login/index.js

```diff
@@ -29,38 +29,95 @@ export const login = {
 
     if (!isInteractive()) {
       throw new InputError('cannot prompt for credentials in a non-interactive shell')
     }
-    const format = new ChalkOrMarkdown(false)
-    const { apiKey } = await prompts({
+    const result = await prompts({
       type: 'password',
       name: 'apiKey',
-      message: `Enter your ${format.hyperlink(
+      message: `Enter your ${terminalLink(
         'Socket.dev API key',
         'https://docs.socket.dev/docs/api-keys'
-      )}`,
+      )} (leave blank for a public key)`,
+      onState: promptAbortHandler
     })
 
-    if (!apiKey) {
-      ora('API key not updated').warn()
-      return
-    }
+    const apiKey = result.apiKey || FREE_API_KEY
 
     const spinner = ora('Verifying API key...').start()
 
-    const oldKey = getSetting('apiKey')
-    updateSetting('apiKey', apiKey)
+    /** @type {import('@socketsecurity/sdk').SocketSdkReturnType<'getOrganizations'>['data']} */
+    let orgs
+
     try {
-      const sdk = await setupSdk()
-      const quota = await sdk.getQuota()
-      if (!quota.success) throw new AuthError()
-      spinner.succeed(`API key ${oldKey ? 'updated' : 'set'}`)
+      const sdk = await setupSdk(apiKey)
+      const result = await sdk.getOrganizations()
+      if (!result.success) throw new AuthError()
+      orgs = result.data
+      spinner.succeed('API key verified\n')
     } catch (e) {
-      updateSetting('apiKey', oldKey)
       spinner.fail('Invalid API key')
+      return
     }
+
+    /**
+     * @template T
+     * @param {T | null | undefined} value
+     * @returns {value is T}
+     */
+    const nonNullish = value => value != null
+
+    /** @type {prompts.Choice[]} */
+    const enforcedChoices = Object.values(orgs.organizations)
+      .filter(nonNullish)
+      .filter(org => org.plan === 'enterprise')
+      .map(org => ({
+        title: org.name,
+        value: org.id
+      }))
+
+    /** @type {string[]} */
+    let enforcedOrgs = []
+
+    if (enforcedChoices.length > 1) {
+      const { id } = await prompts({
+        type: 'select',
+        name: 'id',
+        hint: '\n  Pick "None" if this is a personal device',
+        message: 'Which organization\'s policies should Socket enforce system-wide?',
+        choices: enforcedChoices.concat({
+          title: 'None',
+          value: null
+        }),
+        onState: promptAbortHandler
+      })
+      if (id) enforcedOrgs = [id]
+    } else if (enforcedChoices.length) {
+      const { confirmOrg } = await prompts({
+        type: 'confirm',
+        name: 'confirmOrg',
+        message: `Should Socket enforce ${enforcedChoices[0]?.title}'s security policies system-wide?`,
+        initial: true,
+        onState: promptAbortHandler
+      })
+      if (confirmOrg) {
+        enforcedOrgs = [enforcedChoices[0]?.value]
+      }
+    }
+    updateSetting('enforcedOrgs', enforcedOrgs)
+    const oldKey = getSetting('apiKey')
+    updateSetting('apiKey', apiKey)
```

Comment:
```suggestion
    // MUST DO all updateSetting ON SAME TICK TO AVOID PARTIAL WRITE
    updateSetting('enforcedOrgs', enforcedOrgs)
    const oldKey = getSetting('apiKey')
    updateSetting('apiKey', apiKey)
```